OctaneSdk/Entities/Users/WorkspaceUser.cs

```diff
@@ -23,6 +23,6 @@ namespace MicroFocus.Adm.Octane.Api.Core.Entities
 	/// </summary>
 	public class WorkspaceUser : BaseUserEntity
     {
-        
+        public bool is_api_key { get; set; }
```

Comment:
Generally we used to expose fields in another way , for example 
			{
				return GetStringValue(LANGUAGE_FIELD);
			}
			set
			{
				SetValue(LANGUAGE_FIELD, value);
			}